todo_app/app.py

```diff
@@ -7,50 +8,98 @@
 from flask_dance.contrib.github import github
 from werkzeug.middleware.proxy_fix import ProxyFix
 
+import logging, os, logging.config, time, loggly.handlers
+from loggly.handlers import HTTPSHandler
+from logging import Formatter
+# logging.config.fileConfig('python.conf')
+# logging.config.fileConfig(r'C:\DevOps\M1.ProjectExercise\DevOps-Course-Starter-v2\todo_app\python.conf')
+logging.Formatter.converter = time.gmtime
+logger = logging.getLogger(__name__)
+# logger = logging.getLogger('myLogger')
+
+# logger.info('Test INFO')
+# logger.debug('Test DEBUG')
+# logger.warning('Test WARNING')
+# logger.error('Test ERROR')
+# logger.critical('Test CRITICAL')
+
 def create_app():   
     app = Flask(__name__)
     app.wsgi_app = ProxyFix(app.wsgi_app)
     app.config.from_object(Config())
     app.register_blueprint(blueprint, url_prefix="/login")
+    
+    logging.basicConfig(level=logging.INFO, format='%(asctime)s %(message)s', datefmt='%d %b %Y %H:%M:%S') #(filename='myapp.log',  
+    log_level = os.getenv('LOG_LEVEL')
+    loggly_token = os.getenv('LOGGLY_TOKEN')
+    # print('LOGGLY_TOKEN: ' + loggly_token)
+    app.logger.setLevel(app.config[log_level])
+    app.logger.info('LOG_LEVEL: ' + log_level)
+
+    if loggly_token is not None:
+        handler = HTTPSHandler(f'https://logs-01.loggly.com/inputs/{loggly_token}/tag/todo-app')
+        # print('is not none: ' + loggly_token)
+        # print(handler)
+        handler.setFormatter(
+            # Formatter("[%(asctime)s] %(levelname)s in %(module)s: %(message)s")
```

Comment:
Same as previous comment, just tidy up when you're done using these debug/old lines.